docs/http/http-spans.md

```diff
@@ -244,12 +244,11 @@ For an HTTP client span, `SpanKind` MUST be `Client`.
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `http.resend_count` | int | The ordinal number of request resending attempt (for any reason, including redirects). [1] | `3` | Recommended: if and only if request was retried. |
-| [`server.address`](../general/attributes.md) | string | Host identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. [2] | `example.com` | Required |
+| [`network.peer.address`](../general/attributes.md) | string | Peer address of the network connection - IP address or Unix domain socket name. | `10.1.2.80`; `/tmp/my.sock` | Recommended: If different than `server.address`. |
+| [`network.peer.port`](../general/attributes.md) | int | Peer port number of the network connection. | `65123` | Recommended: If `network.peer.address` is set. |
```

Comment:
Given that `server.port` is *condititonally required*  -- *If not default (`80` for `http` scheme, `443` for `https`)*, I think it would be important to add the same requirement here.

Reply:
I interpret 'recommended' to mean implementations are free to omit it, but it also seems reasonable for the spec to clarify if they do think it is preferable for this port to be there even when it is the default one. @trask what your thoughts on this one?

Reply:
Thanks for the feedback! For now, I prefer not to report `network.peer.port` in .NET despite reporting `network.peer.address`. Justification: https://github.com/dotnet/runtime/issues/92956#issuecomment-1753316516